.go-version

```diff
@@ -0,0 +1 @@
+1.17.5
```

Comment:
Do we need to update this file when upgrading to a new version of Go?

Reply:
Yes! If the version of go changes that is used to build packer, this will need to be updated.